docs/MODULE.bazel.lock

```diff
@@ -621,7 +621,7 @@
     }
   },
   "moduleExtensions": {
-    "@apple_support~1.5.0//crosstool:setup.bzl%apple_cc_configure_extension": {
+    "@@apple_support~1.5.0//crosstool:setup.bzl%apple_cc_configure_extension": {
```

Comment:
Having no experience with bzlmod yet, I don't know if this is expected.  Presumably something to do with Bazel 7.0.0 officially getting released 6 hours ago (🥳)?

Reply:
Ah from: https://github.com/bazelbuild/bazel/releases/tag/7.0.0
```
[Incompatible] All labels in Bazel error messages, log output, Build Event Protocol, etc. are now prefixed with double-at (@@) instead of single-at (@) where applicable, to properly denote that they contain canonical repo names.
```